interface.go

```diff
@@ -309,6 +348,29 @@ func (f *Interface) reloadFirewall(c *config.C) {
 		Info("New firewall has been installed")
 }
 
+func (f *Interface) reloadSendRecvError(c *config.C) {
+	stringValue := c.GetString("listen.send_recv_error", "")
+	if (c.InitialLoad() && stringValue != "") || c.HasChanged("listen.send_recv_error") {
+		switch stringValue {
+		case "always":
+			f.sendRecvErrorConfig = sendRecvErrorAlways
+		case "never":
+			f.sendRecvErrorConfig = sendRecvErrorNever
+		case "private":
+			f.sendRecvErrorConfig = sendRecvErrorPrivate
+		default:
+			if c.GetBool("listen.send_recv_error", true) {
+				f.sendRecvErrorConfig = sendRecvErrorAlways
+			} else {
```

Comment:
The default, if unset, is sendRecvErrorAlways. But if the user set a value that doesn't match, it's sendRecvErrorNever. Is this intended?

I was thinking if an unknown value is set, it should be logged and the default value be applied.